### DISCUSSION TITLE
New install, worked initially now all devices unreachable

### DISCUSSION THREAD

**Discussion by jruben4** [Koenkk/zigbee2mqtt#12277] [General]
New install (unraid docker, Conbee II, HA).  Seemed to be going well, enrolled all my devices (a few Ikea power outlets, a few ikea buttons).  Could turn the switches on/off via the zigbee2mqtt UI as well as HA.  Could see attributes in HA.  

A few hours later the docker was stopped.  Now whenever I restart the container, the UI is working but all devices are basically offline (can't turn them on/off, getting no data back) and the container shuts itself down every hour or less.

Tried 1.23.0 as well as latest-dev and both had same behavior

**Comment by max-rousseau** [comment]
Have a similar problem with 1.25.0-1 on HASSIO with a Sonoff ZigBee 3.0 Plus USB adapter. After restart, all devices are offline.

**Comment by jruben4** [comment]
Fixed switching away from Conbee II.
